requirements.txt

```
@@ -26,4 +26,8 @@ ibm-watson==4.4.0
 websockets==8.1
 webexteamssdk==1.3
 voluptuous==0.11.7
+<<<<<<< HEAD
```

It seems you had git conflicts when you pushed your commit. Not sure if you know how to deal with git conflicts but I'm happy to give you a hand with it if you need, just let me know.

In this case you can quickly fix the issue by removing the `<<<<<<< HEAD`, `=======`, `>>>>>>> 48e61c4ed1b1ccbb146a765170a5ce1d50fa1384`
```suggestion
bitstring==3.1.7 
regex==2020.4.4
```